File: SGA_Api/SGA_Api/Models/Login/LoginResponseDto.cs
namespace SGA_Api.Models.Login
{
    public class LoginResponseDto
    {
        public int Operario { get; set; }
        public string? NombreOperario { get; set; }
        public List<short>? CodigosAplicacion { get; set; }
        public List<string>? CodigosAlmacen { get; set; }
		public List<EmpresaDto> Empresas { get; set; } = new();
		public string Token { get; set; } = string.Empty;

        public string? CodigoCentro { get; set; }
    }
}

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/login/loginResponseDto.kt
package com.example.sga.data.dto.login

data class LoginResponseDto(
    val operario: Int,
    val nombreOperario: String,
    val codigosAplicacion: List<Short>,
    val codigosAlmacen: List<String>,
    val empresas: List<EmpresaDto>,
    val token: String,
    val codigoCentro: String
)
